examples/cloudrun/wait-approval/.pipe.yaml

```diff
@@ -0,0 +1,21 @@
+# Deployment pipeline that contains a manual approval stage.
+apiVersion: pipecd.dev/v1beta1
+kind: CloudRunApp
+spec:
+  notification:
+    pipeline:
+    stages:
+      # Promote new version to receive amount of traffic.
+      - name: CLOUDRUN_PROMOTE
+        with:
+          percent: 10
+      - name: WAIT_APPOVAL
+      # Promote new version to receive all traffic.
+      - name: CLOUDRUN_PROMOTE
+        with:
+          percent: 100
```

Comment:
This pipeline shouldn't be inside the `notification` block.

Reply:
b18b3c0